_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -30989,7 +30989,11 @@
 	},
 /area/hydroponics)
 "bfE" = (
-/mob/living/simple_animal/hostile/retaliate/goat,
+/mob/living/simple_animal/hostile/retaliate/goat{
+	desc = "Pete, the Chef's pet goat from the Caribbean. Not known for their pleasant disposition.";
```

Comment:
Don't varedit things on the map like this.
Instead, create a subtype in the code and have the map file reference this instead.
This makes it easier, in future, to update the object without new map edits.

Reply:
Didn't consider the 'map edit' side, I'll change it back to a subtype.